### PR TITLE
People: Invites: Disable buttons on accept

### LINKED ISSUE #1197: People: Show "accepting" state on logged in accept
Currently, clicking the the "Accept" button on the logged in accept doesn't seem to do anything. We should probably follow the signup form's example (`/start/account/user`) and disable buttons and change the text.

### PR DESCRIPTION
**Pull request by lezama**
Closes #1197 

cc @ebinnion @roccotripaldi for an early review

### CODE REVIEW

**Comment by ebinnion** [comment]
From a UX perspective I like that we're now showing the user that something is in progress.

However, when I tried to accept an invite while logged in, I was never redirected away from the invite accept page:

screen shot 2

File: client/my-sites/invites/invite-accept-logged-in/index.jsx [hunk 1]
```diff
@@ -40,11 +51,11 @@ export default React.createClass( {
...
 					</div>
 					<div className="invite-accept-logged-in__button-bar">
-						<Button onClick={ this.props.decline }>
+						<Button onClick={ this.props.decline } disabled={ this.state.submitting }>
 							{ this.translate( 'Decline', { context: 'button' } ) }
 						</Button>
-						<Button primary onClick={ () => acceptInvite( this.props.invite ) } href={ this.props.redirectTo } >
-							{ this.translate( 'Join', { context: 'button' } ) }
+						<Button primary onClick={ this.accept } disabled={ this.state.submitting }>
+							{ this.state.submitting ? this.translate( 'Joining…', { context: 'button' } ) : this.translate( 'Join', { context: 'button' } ) }
```
Comment (ebinnion) [inline client/my-sites/invites/invite-accept-logged-in/index.jsx]: Maybe we could make this a shorter line by rewriting as:

```js
{ 
	this.state.submitting 
	? this.translate( 'Joining…', { context: 'button' } ) 
	: this.translate( 'Join', { context: 'button' } ) 
}
```

Comment (lezama) [inline client/my-sites/invites/invite-accept-logged-in/index.jsx]: :+1:

**Comment by ebinnion** [comment]
LGTM. :shipit:

**Pull request merged**